pulumi/infra/ui.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional
+
+import pulumi_aws as aws
+from infra import util
+from infra.util import IS_LOCAL
+
+import pulumi
+
+
+class UI(pulumi.ComponentResource):
+    """ Represents the web GUI for Grapl."""
```

Comment:
We have a lot of terms for this - Engagement Edge, Engagement UX, etc - and I'd prefer to keep it at 2 instead of 3 if possible!

Reply:
I'll call it `EngagementUX`.

Reply:
It wouldn't be a bad idea to revisit the Grapl UI name at some point. The word "Engagement" is pretty dated, the UI does a lot more than just Engagements. Created a ticket for this as well: https://github.com/grapl-security/issue-tracker/issues/335